build.sbt

```diff
@@ -63,15 +62,77 @@ lazy val checkerLibrary = (projectMatrix in file("checker-library"))
 lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(name := "akka-serializability-checker-plugin")
   .settings(commonSettings)
-  .settings(libraryDependencies ++= {
-    virtualAxes.value
-      .collectFirst { case x: ScalaVersionAxis => x.value }
-      .map {
-        case "2.13" => scalaPluginDeps213
-        case "2.12" => scalaPluginDeps212
-      }
-      .getOrElse(Seq.empty)
-  })
-  .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
+  .settings(
+    libraryDependencies ++= {
+      virtualAxes.value
+        .collectFirst { case x: ScalaVersionAxis => x.value }
+        .map {
+          case "2.13" => scalaPluginDeps213
+          case "2.12" => scalaPluginDeps212
+        }
+        .getOrElse(Seq.empty)
+    },
+    libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test, betterFiles % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val LocalMavenResolverForSbtPlugins = {
+  // remove scala and sbt versions from the path, as it does not work with jitpack
+  val pattern = "[organisation]/[module]/[revision]/[module]-[revision](-[classifier]).[ext]"
+  val name = "local-maven-for-sbt-plugins"
+  val location = userHome / ".m2" / "repository"
+  Resolver.file(name, location)(Patterns().withArtifactPatterns(Vector(pattern)))
+}
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dump-event-schema"))
```

Comment:
Hmmm I've probably brought that up previously... both `sbt-dump-event-schema` and `sbt-dump-event-schema-plugin` correspond to a **plugin** of a kind...

Reply:
So it's not justfied to add `-plugin` suffix to just one of them and not the other. I'd suggest `dump-event-schema-sbt-plugin` and `dump-event-schema-compiler-plugin` (plus update the lazy val names accordingly)

Reply:
Yea, renaming compiler plugin is a good idea but sbt plugin is another story.
According to sbt naming guidelines, sbt plugins should have a `sbt-` prefix.